superset-frontend/src/dashboard/components/menu/PopoverDropdown.tsx

```diff
@@ -75,20 +71,30 @@ const MenuItem = styled(Menu.Item)`
   }
 `;
 
-class PopoverDropdown extends React.PureComponent {
-  constructor(props) {
+interface HandleSelectProps {
+  key: any;
```

Comment:
this is probably a string or `string | int`?

Reply:
I tried to look for when the `key` is used to determine the type. Likely, this is used by antd and the type is `React.Key`.